tests/src/common/Wsk.scala

```diff
@@ -641,11 +646,12 @@ class WskNamespace()
     }
 }
 
-class WskPackage()
+class WskPackage(override val usePythonCLI: Boolean = true) 
     extends RunWskCmd
     with ListOrGetFromCollection
     with DeleteFromCollection {
     override protected val noun = "package"
+    override def baseCommand = Wsk.baseCommand(usePythonCLI)
```

Comment:
I suggest also making the sealed trait's property abstract (that way there can't be confusion)

```
diff --git a/tests/src/common/Wsk.scala b/tests/src/common/Wsk.scala
index 08f3bba..a3a3b6c 100644
--- a/tests/src/common/Wsk.scala
+++ b/tests/src/common/Wsk.scala
@@ -787,7 +787,7 @@ object Wsk {
 
 sealed trait RunWskCmd {
 
-    val usePythonCLI: Boolean = true
+    val usePythonCLI: Boolean
 
```